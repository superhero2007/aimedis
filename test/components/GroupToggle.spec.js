import React from 'react';
import { mount } from 'enzyme';

import GroupToggle from 'components/GroupToggle';

function setup() {
  return mount(<GroupToggle />);
}

describe('GroupToggle', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.find('.app__group__toggle')).toExist();
    expect(wrapper.find('.app__group__toggle__button')).toExist();
    expect(wrapper.find('.app__group__toggle__button__team')).toExist();
    expect(wrapper.find('.app__group__toggle__button__board')).toExist();
    expect(wrapper.find('.app__group__toggle__content')).toExist();
  });
});
