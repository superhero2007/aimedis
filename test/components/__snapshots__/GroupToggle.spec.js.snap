// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GroupToggle should render properly 1`] = `"<div class=\\"app__group__toggle\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__button\\"><div class=\\"app__group__toggle__button__team active\\"><span>TEAM</span></div><div class=\\"app__group__toggle__button__board\\"><span>BOARD</span></div></div><div class=\\"app__group__toggle__content\\"><div class=\\"row\\"><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__last\\"><div class=\\"app__group__toggle__item__last__image\\"><img src=\\"\\" alt=\\"No asset\\"></div><p class=\\"app__group__toggle__item__last__title\\">Join our team</p><a href=\\"/career\\" class=\\"btn btn__secondary app__group__toggle__item__last__subtitle\\">SEE ALL POSITIONS</a></div></div></div></div></div></div></div></div></div>"`;

exports[`GroupToggle should render properly 2`] = `"<div class=\\"app__group__toggle\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__button\\"><div class=\\"app__group__toggle__button__team\\"><span>TEAM</span></div><div class=\\"app__group__toggle__button__board active\\"><span>BOARD</span></div></div><div class=\\"app__group__toggle__content\\"><div class=\\"row\\"><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"#\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"></a><a href=\\"#\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"></a></div></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"animated\\" style=\\"animation-duration: 1s; opacity: 0;\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__last\\"><div class=\\"app__group__toggle__item__last__image\\"><img src=\\"\\" alt=\\"No asset\\"></div><p class=\\"app__group__toggle__item__last__title\\">Join our team</p><a href=\\"/career\\" class=\\"btn btn__secondary app__group__toggle__item__last__subtitle\\">SEE ALL POSITIONS</a></div></div></div></div></div></div></div></div></div>"`;
